### PR TITLE
Mock exchange

### PR DESCRIPTION
**Pull request by abushev**
Ways to execute 
python3 mock-x.py --requests=data.json --concurrency=100
python3 mock-x.py --limit=10 --requests="empty.json data.json"

### CODE REVIEW

**Comment by venediktov** [comment]
python main.py --requests=data.json --concurrency=100
python main.py --limit=10 --requests="empty.json data.json"

It can maybe go under examples/bidder/mock_exchange/README.md ?

**Comment by venediktov** [comment]
How about renaming main.py to mock-x.py ?

**Comment by venediktov** [comment]
I see by default it does 50% win when 2 files passed to script in example ( second command ). So to get win rate at 10% we have to do this ?

python main.py --limit=10 --requests="empty.json empty.json empty.json empty.json data.json"


perhaps there is even better way to get win rate without array of 4 empty + 1 win json ?

Good job ! We finally have something to adjust win rate and adjust QPS of requests .

**Comment by venediktov** [comment]
Actually 4emty + json gives us 20% win rate . 9empty + json = 10% win ?

**Comment by abushev** [comment]
Its not win but just 10% bid in your example. 
I'm planning to add win percent param and make win notifications. 
Also, I have and idea to make data.json template-like, to generate site, size, floor, geo etc individually for each request. What do you think?

File: examples/bidder/mock_exchange/main.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python2
```
Comment (mrbald) [inline examples/bidder/mock_exchange/main.py]: Python3 GIT handling is much better than that of Python2 (no console polling in non-interactive scripts).

**Comment by venediktov** [comment]
data.json template-like sounds interesting !
Well, you can push this pull request now and do more changes later .

**Comment by venediktov** [comment]
we may add main.py to cmake install , perhaps you need to pick up unique name ? 
 it will be deployed under Release/examples/bin

**Pull request merged**